modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/client/Iroha2Client.kt

```diff
@@ -85,7 +85,7 @@ open class Iroha2Client(
     open val credentials: String? = null,
     open val eventReadTimeoutInMills: Long = 250,
     open val eventReadMaxAttempts: Int = 10,
-    override val coroutineContext: CoroutineContext = Dispatchers.IO + SupervisorJob()
+    override val coroutineContext: CoroutineContext = Dispatchers.IO + SupervisorJob(),
```

Comment:
haven't we agreed to not use this code style?

Reply:
In my opinion it make no sense to resist it (this code style option)

Reply:
as I understand it, it is accepted by most of the community